tests/results.cpp

```diff
@@ -320,7 +320,7 @@ TEST_CASE("results: notifications") {
     }
 
     // Sort in descending order
-    results = results.sort({{0}, {false}});
+    results = results.sort({*table, {{0}}, {false}});
```

Comment:
It's a little unfortunate that this change requires callers of `sort` to restate the table, when both `Results` and `List` already know it.